packages/ui-library/src/components/forms/select/index.test.ts

```diff
@@ -83,4 +83,13 @@ describe('blr-select', () => {
 
     expect(className).to.contain('sm');
   });
+
+  it('is rendering options inside slot', async () => {
+    const element = await fixture(BlrSelectRenderFunction({ ...sampleParams, size: 'sm' }, optionsAsChildren));
+
```

Comment:
This line can be removed for consistency like in packages/ui-library/src/components/feedback/tooltip/index.test.ts